_source/api/logzio-public-api.yml

```diff
@@ -4813,3 +4932,382 @@ definitions:
+        type: integer
+        format: int32
+        minimum: 5
+        maximum: 1440
+        description: Add a waiting period in minutes to space out notifications. (The alert will still trigger but will not send out notifications during the waiting period.)
+        example: 60
+      type:
+        type: string
+        description: Selects the output format for the alert notification. If the alert has no aggregations/group by fields, `JSON` offers the option to send full sample logs without selecting specific fields.
+        enum:
+          - JSON
+          - TABLE
+  AlertQuery:
+    type: object
+    description: Determines when the alert should trigger using any combination of a search query, filters, group by aggregations, accounts to search, and trigger conditions.
+    properties:
+      query:
+        type: string
+        description: >-
+          Provide a Kibana search query written in Lucene syntax. The search query together with the filters select for the relevant logs.
+          
+          
+          Cannot be null - send an asterisk wildcard `*` if not using a search query.
+        default: "*"
+        nullable: false
+        example: type:apache_access
+      filters:
+        $ref: '#/definitions/BoolFilter'
+      groupBy:
+        type: array
+        nullable: true
+        description: Specify 1-3 fields by which to group the results and count them. If you apply a group by operation, the alert returns a count of the results aggregated by unique values.
+        items:
+          type: string
+        maxItems: 3
+        minItems: 0
+      aggregation:
+        $ref: '#/definitions/Aggregation'
+      shouldQueryOnAllAccounts:
+        type: boolean
+        default: true
+        example: false
+        description: Only applicable when the alert is run from the main account. If `true`, the alert runs on the main account and all associated searchable sub accounts. If `false`, specify relevant account IDs for the alert to monitor using the `accountIdsToQueryOn` field.
+      accountIdsToQueryOn:
+        type: array
+        description: Specify Account IDs to select which accounts the alert should monitor. The alert will be checked only on these accounts.
+        items:
+          type: integer
+          format: int32
+          example: 2321
+
+  AlertRecipients:
+    type: object
+    description: Add email addresses and/or endpoint channels to automatically receive notifications with sample data when the alert triggers.
+    properties:
+      emails:
+        type: array
+        description: Array of email addresses to be notified when the alert triggers.
+        items:
+          type: string
+          example: tom.a@logz.io
+      notificationEndpointIds:
+        type: array
+        description: Array of IDs of pre-configured endpoint channels to notify when the alert triggers.
+        items:
+          type: integer
+          format: int32
+          example: 10101
+  SubAlert:
```

Comment:
i see that the name SubAlert appears in the doc, although not required in any of the fields in order to create the api request. 
Is this an issue? should we change the naming asap?